### ISSUE TITLE
iOs 9 devices not discovering shairport-sync 2.4 on raspi

### ISSUE THREAD

**Issue by joonamo** [mikebrady/shairport-sync#132]
I just built and installed the latest version of shairport-sync on my Raspberry Pi because my iOs 9.0.1 devices (iPhone 5 and iPad 4) stopped discovering the vanilla shairport after the OS update. On my Windows machine, TuneBlade discovers and plays music fine on the new shairport-sync installation, but both of my iOs devices still fail to discover the shairport-sync server.

I tried enabling full logging and reading the syslog, but there doesn't seem to be any notification on the log when a device discovers the server. Any ideas where to investigate further?

Here's what shairport-sync -V prints: 2.4-openssl-Avahi-ALSA-metadata

**Comment by joonamo** [comment]
I did some more digging and found that my mac laptop would only discover the server on wired connection. I searched the system log for airplay and for some reason discoveryd reported turning off _airplay when connecting to wifi and turning it back on when I connected the ethernet cable.

I restarted my wireless router and both my laptop and iOS devices immediately discovered the shairport-sync server. Problem solved!

Thanks for continuing where shairport left off! I have plans to use the metadata reporting to display the metadata on some small display or on a led matrix...
